shell/edit/secret/ssh.vue

```diff
@@ -44,7 +44,10 @@ export default {
     update() {
       this.value.setData('ssh-publickey', this.username);
       this.value.setData('ssh-privatekey', this.password);
-      this.value.setData('known_hosts', this.knownHosts);
+
+      if (this.showKnownHosts) {
```

Comment:
With this, the `known_hosts` field will be visible only when editing a secret that already has that value in `data`.
The other solution is enable it by default.

Reply:
I think this is fine